### DISCUSSION TITLE
What are legal and effective ways to use these leads?

### DISCUSSION THREAD

**Discussion by Braydenrud** [merchantprotocol/theadslab#19] [Open Discussion Forum]
Is it as simple as plugging these email addresses into a well thought out email campaign? or is there more to it than that?

Looking at it legally, ethically, and most effective, what are the limitations?

**Comment by jonathonbyrdziak** [comment]
# The Most Effective Way To Use These Leads

Most media buyers today are only capable of getting a 1% conversion rate on average. Up to 8% for really warm audiences.

Imagine being able to generate 30% conversion rates from your media buying. Your CPLs would be insanely low, buying leads for pennies on pennies. Advertisers would flock to you in droves for your "skills".

Don't overcomplicate this offer. You now have a secret weapon that allows you to be the most effective media buyer your clients have ever known.

It's rarely a media buyers responsibility to do anything other than provide the leads... Just because it's SO easy to generate leads for you now, why would you try to complicate it by emailing the lead?

* * *

If you research online you'll find that there are little to no laws that even govern this space. It's still truly the wild west and regulation will make this far more difficult in the future.

But as for today you are free to email whomever you choose, regardless of where you received the data.

## CAN-SPAM LAWS

I'm no lawyer and this isn't legal advice, so don't count it as such. If you do significant research online you'll find that CAN-SPAM does not prevent you from email anybody. You are free to email businesses and consumers alike.

Here are a few of the things CAN-SPAM does require of you:
- You must not obfuscate the network path the email travels (no proxies)
- You must not obfuscate the business or person or entity the email is coming from (no fake accounts)
- You must not obfuscate the message in the email (that's an odd one...)
- You must state that the email is an advertisement. (you can do this in the fine-print at the bottom)
- While you don't have to provide an unsubscribe link, you do need to honor any requests by the recipients to stop sending emails.
- I also believe that you must include your address in the emails, but that might just a method used to show that one is not hiding the origin. 

It's up to you to validate these statements.

## IS IT ETHICAL?

Well that depends who you ask. There's only one good book, but not everybody follows it and it doesn't discuss emails...

Most small companies have taken the stance that they will not email people they don't know.. and then they do in the ordinary course of their business.

The larger companies like Facebook frown on you for using other peoples data (against their ad terms) but then do it themselves behind the spotlights (the facebook shadow profiles).

Most enterprise companies use this exact method to target new markets with their advertising. There's even some technology we haven't made public which is even more powerful. Think about when you're talking to a friend about something new and then the next time you sit down at the TV or computer or open your phone the first product you see is what you were just talking about...

So is it ethical? Probably frowned upon in the public light, but everybody is doing it behind the scenes either directly or indirectly. You be the judge.
